Remotion/Web/Core/ExecutionEngine/WxeParameterAttribute.cs

```diff
@@ -92,7 +92,7 @@ public int Index
 
     public bool Required
     {
-      get { return _required.Value; }
+      get { return _required!.Value; }
```

Comment:
I think that's a bug and should be false if null. please check semantics / add bugfix todo

Reply:
As the instance should be fetched through `Remotion.Web.ExecutionEngine.WxeParameterAttribute.GetAttribute`, `_required` should never be `null` here.

Reply:
Returning false when `_required` is `null` would be weird as the default value is different for value & reference types, which is missing information as there is no associated property info. Shouldn't `Required` return `bool?` then?

Reply:
We could, which defers the default to the rest of our code. Question: What does the DependDB say about the property usage downstream?

Reply:
No usages.